import.py

```diff
@@ -0,0 +1,281 @@
+import os
+import subprocess
+import sys
+
+# from django.contrib.gis.utils.ogrinspect import ogrinspect
```

Comment:
Good catch - there's also a line of running code that's no longer necessary because I stopped using ogrinspect.  I have removed it in commit https://github.com/missoula-ready/disaster-preparedness/commit/87c064dd200a1727e7a039f07f8d2f01c8be832a